var/spack/repos/builtin/packages/google-crc32c/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class GoogleCrc32c(CMakePackage):
+    """CRC32C implementation with support for CPU-specific acceleration instructions."""
+
+    homepage = "https://github.com/google/crc32c"
+    git      = "https://github.com/google/crc32c"
```

Comment:
It's better to download via URL so we can checksum the tarball:
```suggestion
   url = "https://github.com/google/crc32c/archive/refs/tags/1.1.2.tar.gz"
```

Reply:
Done.